Lib/pydoc.py

```diff
@@ -2247,8 +2247,8 @@ def log_message(self, *args):
 
     class DocServer(http.server.HTTPServer):
 
-        def __init__(self, port, callback):
-            self.host = 'localhost'
+        def __init__(self, host, port, callback):
+            self.host = host
```

Comment:
If this class is considered part of the module’s public API, this change would break compatibility.  Alternative ways to achieve the same result could be:

* add host after the existing params, with default value to keep compat
* allow the existing port to be a (host, port) tuple